pixeltable/globals.py

```diff
@@ -405,6 +407,10 @@ def create_replica(destination: str, source: str | catalog.Table) -> Optional[ca
         destination: Path where the replica will be created. Can be either a local path such as `'my_dir.my_table'`, or
             a remote URI such as `'pxt://username/mydir.my_table'`.
         source: Path to the source table, or (if the source table is a local table) a handle to the source table.
+        bucket_name: The name of the pixeltable cloud-registered bucket to use to store replica's data.
+            If no `bucket_name` is provided, the default Pixeltable storage bucket will be used.
+        is_public: If True, replica will be shared publicly. If false, replica will remain private.
```

Comment:
Maybe instead of `is_public: bool`, make this `access: Literal['public', 'internal']` or some such. Would be more future-proof as we plan to provide more nuanced access policies in the future.